go/service/chat_local.go

```diff
@@ -59,12 +59,98 @@ func (h *chatLocalHandler) GetThreadLocal(ctx context.Context, arg keybase1.GetT
 	return h.unboxThread(ctx, boxed, arg.ConversationID)
 }
 
+func retryUpToNTimesUntilNoError(n int, action func() error) (err error) {
```

Comment:
Is there some place else that this kind code exists in some KB repo? Maybe the RPC repo? Either way, this should have some sort of timed wait in here I would think.